src/wilson/dom.cljs

```diff
@@ -46,13 +47,21 @@
       ((by-key sort-fns) by-key rows)
       ((:default sort-fns) by-key rows)))
 
+(defn parse-td-data
+  [x]
+  (if (not (or (vector? x) (string? x)))
+    (pr-str x)
+    x))
```

Comment:
Getting rid of the not and switching the order of the clauses might make this easier to read. Alternatively; if-not instead of if